lib/promiseFtp.coffee

```diff
@@ -106,7 +105,7 @@ class PromiseFtp
       
     # methods listed in otherPrototypeMethods, which don't get a wrapper
     
-    @connect = (options) -> Promise.try () ->
+    @connect = (options) -> Promise.resolve().then () ->
```

Comment:
Yes, there is no effective difference between `Promise.try () ->` and `Promise.resolve().then () ->` except readability.  I have since seen some things indicating performance of Bluebird is better than native promises, however, so I am slightly regretting the removal of Bluebird.  However, since this is a fairly lightweight library, the difference is probably small, and I haven't had time to consider putting Bluebird back in.